Assignment2/src/Assignment2/Example2.java

```diff
@@ -0,0 +1,47 @@
+package Assignment2;
+import java.util.LinkedList;
+import java.util.List;
+public class Example2 {
+	public static String destCity(List<List<String>> paths)
+	{
```

Comment:
follow all the instruction in all below program

Reply:
Yes